### ISSUE TITLE
Change 'form_for' to 'form_to'

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/lucky#967]
I think `form_for` reads super strangely, especially since you pass it an action, not an operation

I'm thinking  `form_to` might make more sense

```crystal
form_to SignUps::Create do
end

# Instead of
form_for SignUps::Create do
end
```

One thing I *don't* like about this is it may not be clear that this is won't accept strings. You'd use regular `form` if you just want a custom form (for example to post to a non-Lucky app). I think that's probably fine though. We can raise a helpful error if a String is passed in.

Also maybe neither is clear! Open to more suggestions

cc @edwardloveall @jwoertink

**Comment by jwoertink** [comment]
hmm.. It may because I'm used to seeing `form_for` from rails and just using it in Lucky, but it doesn't feel weird to me. I wonder if anyone else has ran in to that. I don't think `form_to` would be bad, but I'm on the fence if it's necessary.

**Comment by paulcsmith** [comment]
If it isn't a clear win we'll keep it as-is :) Curious to hear what others think or if someone has a better alternative. This is super low priority though so if it stays the same no biggie

**Comment by edwardloveall** [comment]
Y'all got me thinking about it 😄 

TL;DR I read it like `form_for` talks about the action a user takes, and `form_to` is more for the developer.

I think the `form_to` is a nicer phrase when you think about it from the perspective of how data is passed from one place _to_ another, as opposed to data being _for_ a particular purpose.

And _for_ is nice because it remains so agnostic. This can be a form for resetting your password, or for inputting a turn for an online game, or for writing a blog post. It's much more user-centric.

Neither of them bother me, and I can see the rationale for both. What about `form_for` do you think reads poorly @paulcsmith?

**Comment by paulcsmith** [comment]
The more I read it and the more I think about the arguments for `form_for` the more I'm ok with it :) I was just bringing it up in case one was clearer, but I think we can close this now. I kind of like `form_for`  even more now :P
